docs/aws/audit/iamcompliance/rules/root_account_access_keys.mdx

```diff
@@ -23,6 +23,214 @@ HIPAA, ISO27001, PCIDSS, GDPR, NIST, SOC2, CISAWS, CBP, HITRUST, AWSWAF, NISTCSF
+       except (NoCredentialsError, PartialCredentialsError) as e:
+           print(f"Error: {e}")
+
+   # Run the check
+   check_root_access_keys()
+   ```
+
+This script will help you ensure that the root account does not have any access keys, thereby preventing potential security risks associated with root account access keys.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console as an IAM user or role.
+2. In the navigation pane, choose "Users".
+3. In the "User Name" list, choose the name of the intended user which will take you to the "Summary" page.
+4. In the "Security Credential" section, look for "Access Keys". If any access keys (Access Key ID and Secret Access Key) are listed, it means the root account has access keys.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Once you have AWS CLI installed and configured, you can proceed to the next steps.
+
+2. Open your terminal or command prompt and run the following command to list all the users in your AWS account:
+
+   ```
+   aws iam list-users
+   ```
+   This command will return a list of all the IAM users in your AWS account.
+
+3. For each user in the list, run the following command to list the access keys associated with that user:
+
+   ```
+   aws iam list-access-keys --user-name <username>
+   ```
+   Replace `<username>` with the name of the user. This command will return a list of all the access keys associated with the specified user.
+
+4. If the root account is listed in the output of the first command, and it has access keys associated with it (as shown in the output of the third command), then the root account has access keys, which is a misconfiguration. The root account should not have access keys associated with it, as this can pose a security risk.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. First, you need to install the AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    aws_session_token='SESSION_TOKEN',
+)
+```
+
+3. Now, you can use the IAM client to list all the access keys associated with your root account:
+
+```python
+iam = session.client('iam')
+
+response = iam.list_access_keys(UserName='root')
+
+access_keys = response['AccessKeyMetadata']
+```
+
+4. Finally, you can check if any access keys are associated with your root account:
+
+```python
+if access_keys:
+    print("Root account has access keys")
+else:
+    print("Root account does not have access keys")
+```
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target="_blank">Click here for more details</a></b>